cms/static/js/factories/login.js

```diff
@@ -54,4 +54,6 @@ define(['jquery.cookie', 'utility', 'common/js/components/utils/view_utils'], fu
             });
         });
     };
+
+    LoginFactory(...window.pageFactoryArguments["LoginFactory"]);
```

Comment:
@arizzitano: What's the best way for this to fail if `window.pageFactoryArguments` hasn't been initialized?